src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/get-started/architecture.mdx

```diff
@@ -46,17 +46,19 @@ The extension is a small application that integrates tightly with the AWS Lambda
 both the time it takes your telemetry to arrive at New Relic, and the impact that instrumentation has on your
 function's latency and throughput.
 
-### New Relic's backend
+See more about [our Lambda extension](https://github.com/newrelic/newrelic-lambda-extension).
 
-The New Relic service receives your telemetry, processes it into `AwsLambdaInvocation`, `AwsLambdaInvocationError`,
-`Span`, and custom events, and stores all that in our telemetry database: NRDB.
+### New Relic's backend [#backend]
 
-### New Relic One
+The New Relic service receives your telemetry, processes it into [`AwsLambdaInvocation`](https://docs.newrelic.com/attribute-dictionary/?event=AwsLambdaInvocation), [`AwsLambdaInvocationError`](https://docs.newrelic.com/attribute-dictionary/?event=AwsLambdaInvocationError),
+[`Span`](https://docs.newrelic.com/attribute-dictionary/?event=Span), and custom events, and stores all that in our telemetry database: NRDB.
+
+### Lambda UI in New Relic One [#lambda-ui]
 
 Lambda functions aren't quite like traditional services, so the experience of managing them is a little different
-from the classic APM experience. Inside New Relic One, Lambda functions have a custom UI, which quickly surfaces the
-most important information about your function, and integrates closely with our logging and distributed tracing
-features. Backed by NRDB and NRQL, you can also write custom dashboards and alerts for your functions.
+from the classic APM experience. Inside [New Relic One](/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one/), Lambda functions have a custom UI, which quickly surfaces the most important information about your function, and integrates closely with our logging and distributed tracing features. 
+
+Backed by NRDB and NRQL, you can also write custom [dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/) and [alerts](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/introduction-alerts/) for your functions.
 
 ## The legacy CloudWatch path [#cloudwatch]
 
```

Comment:
For the legacy CloudWatch part, we should probably update the log ingestion function name and link to the repo, which is here: https://github.com/newrelic/aws-log-ingestion

Reply:
Thanks @mrickard! What's the name of the new function? aws-log-ingestion?

Reply:
Done!